subsys/bootloader/bl_validation/bl_validation.c

```diff
@@ -140,18 +140,20 @@ validation_info_find(u32_t start_address, u32_t search_distance)
 
 
 #ifdef CONFIG_SB_VALIDATE_FW_SIGNATURE
-static bool validate_signature(u32_t fw_src_address,
-				const struct fw_info *fwinfo,
+static bool validate_signature(const u32_t fw_src_address,
+				const u32_t fw_size,
```

Comment:
It looks like the alignment is a bit off here.

Reply:
I usually align with tabs only. I managed to remove a line here though.

Reply:
AFAIK the zephyr code standard is to align with tabs as far as you can, and then space align the rest.

Reply:
Done